index.html

```diff
@@ -32,13 +34,124 @@ <h5>
+      <li><a href="#"><i class="fa-brands fa-linkedin-in socialMedia"></i></a> </li>
+      <li><a href="#"><i class="fa-solid fa-m socialMedia"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-github socialMedia"></i></a></li>
+      <li><a href="#"><i class="fa-regular fa-hand-peace socialMedia"></i></a></li>
     </ul>
   </section>
+  <section class="work">
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio0.png" alt="" srcset="">
+      <h2>Tonic</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio1.png" alt="Project_Image" srcset="">
+      <h2>Multi-Post <br>
+      Stories</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio2.png" alt="Project_Image" srcset="">
+      <h2>Tonic</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio3.png" alt="Project_Image" srcset="">
+      <h2>Multi-Post<br>
+      Stories</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+  </section>
+  <section class="about_section">
+        <h2>About Myself</h2>
```

Comment:
- [ ] Kindly work on the typography and structure of the About Myself section as requested by the first reviewer. I would break it down for you so that you have an idea of what you have to do for each element. For this h2 element, it would be nice to use the same `font-size`, `color` and `font-weight.` Also, notice the structure of the element. It is two-layered and not one-layered as you have it. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219477070-35ccb29b-a82b-4cf2-b74c-ad52b33e27be.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/219477141-f9fd2877-c6b7-4d27-aba6-6c00ae5d5816.png)